### PR TITLE
3.11 Translation - Romanian

### PR DESCRIPTION
**Pull request by leptun**
This PR implements the Romanian language in the community section. It is based on #3345 (exceeded the language count of the bootapp), #3348 (needed a longer translation for "Unload filament") and is related to #3343

@DRracer @3d-gussner @hauzman fyi.

@todo:
- [x] Merge prerequisite PRs
- [x] Regenerate PO files

PFW-1283

### CODE REVIEW

File: lang/lang_en_ro.txt [hunk 1]
```diff
@@ -240,7 +240,7 @@
 
 #MSG_EJECT_FILAMENT c=16
 "Eject filament"
-"Scoate filament"
+"Scoate filamentul"
```
Comment (leptun) [inline lang/lang_en_ro.txt]: Nu incape. D-aia l-am lasat asa.

File: lang/lang_en_ro.txt [hunk 2]
```diff
@@ -340,7 +340,7 @@
 
 #MSG_FILAMENT_SENSOR c=20
 "Filament sensor"
-"Senzorul de filament"
+"Senz. de filament"
```
Comment (leptun) [inline lang/lang_en_ro.txt:343]: Asta incape asa cum e acum. Nu prea are sens sa fie facut mai scurt

File: lang/lang_en_ro.txt [hunk 3]
```diff
@@ -528,7 +528,7 @@
 
 #MSG_LOAD_FILAMENT c=17
 "Load filament"
-"Incarca fil."
+"Incarca filamentul"
```
Comment (leptun) [inline lang/lang_en_ro.txt]: Nu incape. D-aia l-am lasat asa.

File: lang/lang_en_ro.txt [hunk 4]
```diff
@@ -1128,7 +1128,7 @@
 
 #MSG_UNLOAD_FILAMENT c=16
 "Unload filament"
-"Scoate filament"
+"Descarca filament"
```
Comment (leptun) [inline lang/lang_en_ro.txt:1131]: Nu incape. D-aia l-am lasat asa. Stiu ca suna putin tampit, dar nu am gasit un cuvant care sa aiba sens si sa incapa

**Review by leptun — COMMENTED** [review]
@hauzman Am facut niste comentarii la ultimul commit. Alea fara comment sunt ok. Cele cu comment trebuie schimbate

File: lang/lang_en_ro.txt (hunk 2 above)
Comment (Hauzman) [reply lang/lang_en_ro.txt:343]: Incape dar cand face testarea si rotita ce se invarte da peste "nt" din capat

File: lang/lang_en_ro.txt (hunk 1 above)
Comment (Hauzman) [reply lang/lang_en_ro.txt:243]: Pe printer incape

File: lang/lang_en_ro.txt (hunk 3 above)
Comment (Hauzman) [reply lang/lang_en_ro.txt]: In piza de mai jos se vede prumul rand de sus

**Comment by DRracer** [comment]
@leptun do you know why do we have:
```
#MSG_LOAD_FILAMENT c=17
#MSG_UNLOAD_FILAMENT c=16
```
Shouldn't these two be set the same? Or do we use the messages elsewhere as well (and thus we need each of them limited to a different number of characters)?

File: lang/lang_en_ro.txt (hunk 1 above)
Comment (leptun) [reply lang/lang_en_ro.txt]: Ba nu. Incape acolo, dar mai e folosit acelasi mesaj si in alte locuri unde sunt maxim 16 caractere

File: lang/lang_en_ro.txt (hunk 2 above)
Comment (leptun) [reply lang/lang_en_ro.txt:343]: Ouf. Vad acum. E o greseala la limita de caractere.

@3d-gussner Please fix the `#MSG_FILAMENT_SENSOR c=20`. It does not exist in the code. It should be `MSG_SELFTEST_FILAMENT_SENSOR` with c=17

File: lang/lang_en_ro.txt (hunk 3 above)
Comment (leptun) [reply lang/lang_en_ro.txt]: Aceeasi problema ca la Scoate filamentul. E folosit si la multimaterial mesajul unde pune codul un numar dupa mesaj si nu incape pe ecran acolo

File: lang/lang_en_ro.txt (hunk 4 above)
Comment (leptun) [reply lang/lang_en_ro.txt:1131]:
```
const char MSG_UNLOAD_FILAMENT[] PROGMEM_I1 = ISTR("Unload filament"); ////Number 1 to 5 is added behind text e.g. "Unload filament" c=16
```
Asta e putin dubios in cod. Nu mai e adevarat acel comment.

@3d-gussner Can you please investigate the usage of this message? It looks as if it's used (was intended to be used?) with the SNMM to replace some older stupid translations, but the limit seems a bit arbitrary to me in the context of the MK2/3 + MMU2. I think the limit should be c=18, but I may be wrong.

**Comment by leptun** [comment]
@DRracer Yes, I also noticed the issue as well. There is some problem with these load/unload/eject messages because of the old SNMM code and because of the MMU2 using some of them with a number after the string. These should be fixed in a different PR.

File: lang/lang_en_ro.txt (hunk 3 above)
Comment (Hauzman) [reply lang/lang_en_ro.txt]: > OK modific inapoi, o sa mai incerc odata cu mmu.

Comment (Hauzman) [reply lang/lang_en_ro.txt]: La partea asta spuneai

Comment (leptun) [reply lang/lang_en_ro.txt]: Da. Load filament, eject filament si cut filament au numar dupa ele, asa ca trebuie sa fie mai scurte putin

File: lang/lang_en_ro.txt (hunk 1 above)
Comment (leptun) [reply lang/lang_en_ro.txt]: Asa cum arata in ultima poza nu e ok. Te rog sa il lasi mai scurt cum era inainte asa incat sa incapa si aici, si in Main Menu

File: lang/lang_en_ro.txt (hunk 3 above)
Comment (Hauzman) [reply lang/lang_en_ro.txt]: @leptun Am Modificat deja in "Incarca filament" numa poza era facuta inainte sa o modific :) si nu am mai dat push la modificarile noi  inca.

File: lang/lang_en_ro.txt (hunk 2 above)
Comment (leptun) [reply lang/lang_en_ro.txt:343]: I will mark this as resolved as soon as @3d-gussner sees the message

File: lang/lang_en_ro.txt (hunk 4 above)
Comment (leptun) [reply lang/lang_en_ro.txt:1131]: This one is also ok. The c limit needs to be updated in the upstream for the error during build to go away. I will mark this as resolved as soon as @3d-gussner sees it

**Comment by leptun** [comment]
@hauzman Pe langa cele doua threaduri care au ramas deschise momentan, ultimul commit cred ca a rezolvat toate problemele ramase cu traducerea asta.

**Comment by Hauzman** [comment]
> @hauzman Pe langa cele doua threaduri care au ramas deschise momentan, ultimul commit cred ca a rezolvat toate problemele ramase cu traducerea asta.
@leptun momentan nu am vazut nici eu alt ceva, o sa dau ceva print sa faca si am sa ma mai uit peste el in xaz ca gasesc ceva modific

**Review by andreitudorp — APPROVED** [review]
Looks good!

**Comment by leptun** [comment]
@Hauzman Any update over the weekend?

**Comment by Hauzman** [comment]
> @Hauzman Any update over the weekend?

@leptun No I have made a 6H print yesterday and all seems Ok.
I have printed with MMU and no issues except a few y crashes but this was before.

**Comment by leptun** [comment]
Ok. I'm marking this as ready for review.
The prerequisite PR is also ready for review. The merge order is clear.
@Hauzman Please approve the PR.

**Review by Hauzman — APPROVED** [review]

**Review by Hauzman — APPROVED** [review]

**Review by 3d-gussner — APPROVED** [review]
Approved by 2 translators. So also an approval from me.

**Pull request merged**